packages/integration-sdk-core/src/data/createIntegrationEntity.ts

```diff
@@ -10,7 +10,7 @@ import { parseTimePropertyValue } from './converters';
 import { validateRawData } from './rawData';
 import { assignTags, ResourceTagList, ResourceTagMap } from './tagging';
 
-const SUPPORTED_TYPES = ['string', 'number', 'boolean'];
+const SUPPORTED_TYPES = ['string', 'number', 'boolean', 'array', 'undefined'];
```

Comment:
Wouldn't this allow Arrays of arrays? I don't think we need to allow array here since we have a check for that already.
```suggestion
const SUPPORTED_TYPES = ['string', 'number', 'boolean',  'undefined'];
```

Reply:
Also isn't typeof [] === 'object' ?

Reply:
> Also isn't typeof [] === 'object' ?

That's why we have Array.isArray

Reply:
https://github.com/JupiterOne/sdk/pull/1002/files#diff-4d79fa9d80062cd821c28d4014d267e94d76e24d49610be304dc8513b06981b8R199

Array are also allowed.
You are right about array inside arrays.